python-package/README.rst

```diff
@@ -142,6 +141,17 @@ By default, installation in environment with 32-bit Python is prohibited. Howeve
 
 It is **strongly not recommended** to use this version of LightGBM!
 
+Install from `conda-forge <https://anaconda.org/conda-forge/lightgbm>`_
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

Comment:
```suggestion
Install from `conda-forge channel <https://anaconda.org/conda-forge/lightgbm>`_ Using ``conda``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```

Reply:
@StrikerRUS I previously recommended removing the "Using" part (https://github.com/microsoft/LightGBM/pull/3544#discussion_r554287324). I think it makes the title more verbose but doesn't provide any additional information

Reply:
Ah, I see, OK!

Then please ignore my previous suggestion. And I think it will make sense to remove the "Using" part from `Install from PyPI Using pip` heading above.

Reply:
Thanks for the discussion. I'm unsure on what the suggestion is here. You are welcome to push to my branch.

Reply:
sorry for the confusion. I just pushed https://github.com/microsoft/LightGBM/pull/3544/commits/5a91ca1a7df340c383f0d588cefb451c7b17f0a7 to resolve this thread.

Reply:
and https://github.com/microsoft/LightGBM/pull/3544/commits/ffd78bdb3a36a839215abbfb6d19e8f48876e0dd to address @StrikerRUS 's suggestion.